c2rust-refactor/src/rewrite/strategy/print.rs

```diff
@@ -252,7 +252,24 @@ impl Splice for Ty {
 
 impl Splice for Stmt {
     fn splice_span(&self) -> Span {
-        self.span
+        // Extend statement span to include attributes on the inner node.
+        //
+        // Transforms like sink_lets set stmt.span = local.span, but attributes are
```

Comment:
You're fixing `sink_lets` below, is this also needed?

Reply:
I'm not exactly sure how that would be done, can you put attributes inside a statement? I thought that was contained at a higher level.